### PR TITLE
fix for ussue 1835, underlying template changes

### PR DESCRIPTION
**Pull request by pineapplespatula**
fix for issue #1835

### CODE REVIEW

**Comment by iwehrman** [comment]
Microsoft Surface Pro, we hardly knew ye.

File: src/static/templates.json [hunk 1]
```diff
@@ -31,13 +31,7 @@
...
         "id": "ANDROID_1080P",
-        "preset": "Android 1080p (1920, 1080)",
-        "width": 1920,
-        "height": 1080
-    },
-    {
-        "id": "MS_SURFACE_PRO",
-        "preset": "Microsoft Surface Pro (1920, 1080)",
+        "preset": "Android 1080p (1080, 1920)",
         "width": 1920,
```
Comment (iwehrman) [inline src/static/templates.json]: I think the Android 1080p width and height values should be swapped.

Comment (pineapplespatula) [inline src/static/templates.json]: I just swapped them in order to match the PS template. #That'sTheFix

**Comment by pineapplespatula** [comment]
@iwehrman back to you

File: src/static/templates.json (hunk 1 above)
Comment (iwehrman) [inline src/static/templates.json]: No, the width and height _properties_ need to be switched, not the values in the strong. When you create an `Android 1080p (1080, 1920)` document you get a document of width 1080 and height 1920, but this file has `width: 1920` and `height: 1080`. Those values are used here:
android

**Comment by pineapplespatula** [comment]
ahh i see .

**Comment by iwehrman** [comment]
:grapes:

**Pull request merged**